Tests/Mocks/StubLogger.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+@testable import Frames
+import CheckoutEventLoggerKit
+
+class StubLogger: CheckoutEventLogging {
+    var logCalled = false
+    var logCallArgs: [Event] = []
+
+    func log(event: Event) {
+        logCalled = true
+        logCallArgs.append(event)
+    }
+
+    var registerMetadataCalled = false
+    var registerMetadataCallArgs: [AddMetadataCallArgs] = []
+
+    func add(metadata: String, value: String) {
+        registerMetadataCalled = true
+        registerMetadataCallArgs.append(AddMetadataCallArgs(metadata: metadata, value: value))
+    }
+
+    func remove(metadata: String) {
```

Comment:
empty methods

Reply:
this method is required to conform to the `CheckoutEventLogging` protocol